docs/framework/windows-workflow-foundation/state-machine-workflows.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "State Machine Workflows"
+description: Learn about creating state machine workflows using the StateMachine activity, with State, Transition, and others, to build state machine workflow programs.
```

Comment:
```suggestion
description: Learn about creating state machine workflows using the StateMachine activity.
```

Reply:
@gewarren - this one comes in under 100 char - and we'll bend that rule - you have final say.

Is this OK:

This article provides an overview of creating state machine workflows using the StateMachine activity.

Reply:
Yes, that sounds great.